kfp/pipeline_generator/example/pipeline_definitions.yaml

```diff
@@ -0,0 +1,24 @@
+pipeline_parameters:
+    name: "noop"
+    description: "Pipeline for noop task"
+    script_name: "noop_transform.py"
+    prefix: ""
+    multi_s3: False
+    compute_func_name: ""
+    compute_func_import: ""
+    component_spec_path: ""
+
+pipeline_common_input_parameters_values:
+    kfp_base_image: "quay.io/dataprep1/data-prep-kit/kfp-data-processing:0.1.1"
+    transform_image: "quay.io/dataprep1/data-prep-kit/noop:0.8.0"
+    s3_access_secret: "s3-secret"
+    image_pull_secret: "prod-all-icr-io"
+    input_folder: "test/noop/input/"
```

Comment:
How is this going to work for multiple inputs/outputs?

Reply:
There is a parameter `multi_s3` that can be set to true, then it will take the execute component of multi. The input parameters can be added to `pipeline_transform_input_parameters` section.